### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by razaghimahdi**
Added DataStore, new solution of data storage

### CODE REVIEW

**Review by rviannaoliveira — APPROVED** [review]

**Review by rviannaoliveira — APPROVED** [review]

**Review by rviannaoliveira — COMMENTED** [review]
This Data Store is encrypted?

**Comment by razaghimahdi** [comment]
before I answer that, let me say I'm so sorry if it took that much long to response. Currently not an option to actually save your data encrypted in datastore, however we can actually set it up to do exactly that. for better understanding check this sample out:
https://github.com/philipplackner/AndroidCrypto/blob/master/app/src/main/java/com/plcoding/androidcrypto/CryptoManager.kt

File: README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ If you have any questions, suggestions let's have it over [Issues](https://githu
 
 - 📦 Safe Storage
   * [Cryptography of sharedpreferences](https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences)
+  * [DataStore(modern data storage solution)](https://developer.android.com/topic/libraries/architecture/datastore)
```
Comment (amorimgabriel) [inline README.md]: Data Store can be used as encrypted but I believe that the best solution is Data Store + CryptoManager or any kind of encryption before store it inside the data store to be secure.

```suggestion
  * [DataStore + Cipher (to secure the data)
  (modern data storage solution)](https://developer.android.com/topic/libraries/architecture/datastore)
```

Comment (amorimgabriel) [reply README.md]: This suggestion is because the target of this repo, is to share content related with security, and ways to work more securely.

Comment (razaghimahdi) [reply README.md]: it depends on  what library u want to encrypt it with, but u r absolutely right

Comment (rviannaoliveira) [reply README.md]: Putting what Amorim brought, for me it's done

**Comment by rviannaoliveira** [comment]
@razaghimahdi after of changes, can you merged, please?

**Comment by razaghimahdi** [comment]
> @razaghimahdi after of changes, can you merged, please?

yes sir, I guess it's merged now :)

**Comment by rviannaoliveira** [comment]
curiosity can you merge or do I have to configure something in the repo?

**Comment by razaghimahdi** [comment]
> curiosity can you merge or do I have to configure something in the repo?

LOL, I mean I pushed :))))))
no merge 😂

**Pull request merged**